GameData/ROCapsules/PartConfigs/GeminiBDB/GeminiEquipmentSection.cfg

```diff
@@ -0,0 +1,388 @@
+				transform = GoldFoil
+			}
+		}
+	}
+
+	MODULE
+	{
+		name = ModuleB9PartSwitch
+		moduleID = meshSwitchNodes
+		switcherDescription = OAMS Nodes
+		switcherDescriptionPlural = OAMS Nodes
+		affectDragCubes = False
+		affectFARVoxels = False
+
+		SUBTYPE
+		{
+			name = None
+			title = None
+			descriptionSummary = No engine attach nodes.
+			descriptionDetail = Who needs forward thrust anyway?
+		}
+		SUBTYPE
+		{
+			name = Two
+			title = 2 Nodes
+			descriptionSummary = Two OAMS thruster attach points at the bottom of the module.
+			descriptionDetail = 
+			defaultSubtypePriority = 1
+			node = OAMS_node1
+			node = OAMS_node2
+			stackSymmetry = 1
+		}
+		SUBTYPE
+		{
+			name = Four
+			title = 4 Nodes
+			descriptionSummary = Four OAMS thruster attach points at the bottom of the module.
+			descriptionDetail = 
+			node = OAMS_node1
+			node = OAMS_node2
+			node = OAMS_node3
+			node = OAMS_node4
+			stackSymmetry = 3
+		}
+	}
+}
+
+//	================================================================================
+//	TAC Life Support Compatibility
+//	================================================================================
+@PART[ROC-GeminiEquipmentSectionBDB]:FOR[ROCapsules]:NEEDS[TacLifeSupport]
+{
+	MODULE
+	{
+		name = TacGenericConverter
+		%converterName = Fuel Cell
+		%StartActionName = Start Fuel Cell
+		%StopActionName = Stop Fuel Cell
+		%conversionRate = 1.0
+		//inputResources = LqdHydrogen, 0.000296379, LqdOxygen, 0.000210317
+		//outputResources = WasteWater, 0.00025967179, true, ElectricCharge, 2.2, true
+		INPUT_RESOURCE
+		{
+			%ResourceName = LqdHydrogen
+			%Ratio = 0.000296379
+		}
+		INPUT_RESOURCE
+		{
+			%ResourceName = LqdOxygen
+			%Ratio = 0.000210317
+		}		
+		OUTPUT_RESOURCE
+		{
+			%ResourceName = WasteWater
+			%Ratio = 0.00025967179
+			%DumpExcess = true
+		}
+		OUTPUT_RESOURCE
+		{
+			%ResourceName = ElectricCharge
+			%Ratio = 2.2
+			%DumpExcess = false // if the batteries are full, we would want the full cell to stop running, right?
+		}
+	}
+	MODULE
+	{
+		name = TacGenericConverter
+		%converterName = LOX-O2
+		%StartActionName = Start Oxygen Generator
+		%StopActionName = Stop Oxygen Generator
+		%conversionRate = 2.0
+		//inputResources = LqdOxygen, 0.0000085058788, ElectricCharge, 0.025
+		//outputResources = Oxygen, 0.006883126, false
+		INPUT_RESOURCE
+		{
+			%ResourceName = LqdOxygen
+			%Ratio = 0.0000085058788
+		}
+		INPUT_RESOURCE
+		{
+			%ResourceName = ElectricCharge
+			%Ratio = 0.025
+		}
+		OUTPUT_RESOURCE
+		{
+			%ResourceName = Oxygen
+			%Ratio = 0.006883126
+			%DumpExcess = false
+		}
+	}
+}
+
+//	================================================================================
+//	Final Pass to Make Sure TAC does not add extra resources
+//	================================================================================
+
+@PART[ROC-GeminiEquipmentSectionBDB]:FOR[zzzRealismOverhaul]
+{
+	!RESOURCE:HAS[~name[Ablator],~name[CharredAblator]],*{}
```

Comment:
Why are we okay with Ablator/CharredAblator being added to this?

Reply:
The TACLS cleanup was probably okay, I was just questioning why the exception for Ablator & CharredAblator.  Th exception makes sense if the part has an integrated heatshield, but we are removing those.